qiita_pet/test/rest/test_study_person.py

```diff
@@ -45,6 +54,11 @@ def test_get_invalid_query_string(self):
                             headers=self.headers)
         self.assertEqual(response.code, 400)
 
+    def test_get_invalid_query_string_2(self):
+        response = self.get('/api/v1/person?affiliation=knight%20lab',
+                            headers=self.headers)
+        self.assertEqual(response.code, 400)
```

Comment:
Nope - the [response code of a missing argument is 400](http://www.tornadoweb.org/en/stable/web.html#tornado.web.MissingArgumentError) :-) I was basically replicating the behavior of that exception with less readable code xD